src/net.h

```diff
@@ -734,8 +735,8 @@ class CNode
     int64_t nNextAddrSend GUARDED_BY(cs_sendProcessing){0};
     int64_t nNextLocalAddrSend GUARDED_BY(cs_sendProcessing){0};
 
-    const bool m_addr_relay_peer;
-    bool IsAddrRelayPeer() const { return m_addr_relay_peer; }
+    bool IsAddrRelayPeer() { return m_addr_relay_peer; }
```

Comment:
```suggestion
    bool IsAddrRelayPeer() const { return m_addr_relay_peer; }
```

This can be `const`, just as it was before.